packages/zoe/src/zoeService/zoe.js

```diff
@@ -79,26 +79,32 @@ const makeZoeKit = (
 
   const feeMintKit = vivifyFeeMint(zoeBaggage, feeIssuerConfig, shutdownZoeVat);
 
+  // XXX can we tell ts that this ensures vatAdminSvcP is non-null?
+  const assertVatAdminSvcP = () => {
+    vatAdminSvcP || Fail`createZCFVat did not get bundleCap`;
```

Comment:
I believe the answer is yes, per https://github.com/Agoric/agoric-sdk/blob/c680b92375748c2608ae22bb26ad9a09d38aca90/packages/assert/src/types.js#L300-L316
```suggestion
  const assertVatAdminSvcP = () => {
    if (!vatAdminSvcP) {
      throw Fail`createZCFVat did not get bundleCap`;
    }
```

Reply:
I tried that, and it works when it's in-line, but doesn't when the conditional is pulled out to a function. Perhaps there's a jsdoc annotation that could be added to the function that would do the job.

Reply:
With @turadg's help, changed it to 
```
  const getActualVatAdminSvcP = () => {
    if (!vatAdminSvcP) {
      throw Fail`createZCFVat did not get bundleCap`;
    }
    return vatAdminSvcP;
  };
```
So the users don't separately assert, instead, they just call `E(getActualVatAdminSvcP()).whatever()`.